test/e2e/network/ingress.go

```diff
@@ -935,3 +939,176 @@ func detectNegAnnotation(f *framework.Framework, jig *e2eingress.TestJig, gceCon
 		framework.ExpectNoError(err)
 	}
 }
+
+var _ = SIGDescribe("Ingress API", func() {
+	f := framework.NewDefaultFramework("ingress")
+	/*
+		Release: v1.19
+		Testname: Ingress API
+		Description:
+		- The networking.k8s.io API group MUST exist in the /apis discovery document.
+		- The networking.k8s.io/v1beta1 API group/version MUST exist in the /apis/networking.k8s.io discovery document.
+		- The ingresses resources MUST exist in the /apis/networking.k8s.io/v1beta1 discovery document.
+		- The networking/ingresses resource must support create, get, list, watch, update, patch, delete, and deletecollection.
```

Comment:
nit: these lines get concatenated into conformance descriptions, so drop the leading `-`:

```suggestion
		The networking.k8s.io API group MUST exist in the /apis discovery document.
		The networking.k8s.io/v1beta1 API group/version MUST exist in the /apis/networking.k8s.io discovery document.
		The ingresses resources MUST exist in the /apis/networking.k8s.io/v1beta1 discovery document.
		The networking/ingresses resource must support create, get, list, watch, update, patch, delete, and deletecollection.
```